src/guides/v2.3/rest/tutorials/bundle-product/create-bundle-products.md

```diff
@@ -0,0 +1,316 @@
+---
+group: rest-api
+title: Create bundle products tutorial
+functional_areas:
+  - Integration
+---
+
+A bundle is a build your own, customizable product. Each bundle product can be either
+[Simple Product](https://docs.magento.com/user-guide/catalog/product-create-simple.html) or [Virtual Product](https://docs.magento.com/user-guide/catalog/product-create-virtual.html)
+
+This tutorial describes how you can use the Magento REST API to create bundle products.
+
+{:.bs-callout-info}
+Install a Magento 2.3 (or later) instance with sample data defines a functional store, called Luma. Use REST client to send calls to Magento , see [Postman](https://www.getpostman.com/). Obtain an admin authorization token. See [How to generate the admin token]({{ page.baseurl }}/rest/tutorials/prerequisite-tasks/create-admin-token.html) for more information. [REST Tutorials]({{ page.baseurl }}/rest/tutorials/index.html) provides additional information about completing any Magento REST tutorial.
```

Comment:
```suggestion
Install a Magento 2.3 (or later) instance with sample data which creates a functional store using the Luma theme. Use REST client to send calls to Magento , see [Postman](https://www.getpostman.com/). Obtain an admin authorization token. See [How to generate the admin token]({{ page.baseurl }}/rest/tutorials/prerequisite-tasks/create-admin-token.html) for more information. [REST Tutorials]({{ page.baseurl }}/rest/tutorials/index.html) provides additional information about completing any Magento REST tutorial.
```

Reply:
as a configurable product, I will change & push the changes @BarnyShergold

Reply:
Above grammatical changes updated @BarnyShergold